### PR TITLE
Input

### PR DESCRIPTION
**Pull request by ckahub**
#Input
The input field can be specified using where a user can enter data. The input tag is used within form element to declare input controls that allow users to input data. An input field can be of various types depending upon the attribute type. The Input tag is an empty element which only contains attributes
input01
INPUT02
.

### CODE REVIEW

File: Components/Input/input.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            <ul class="right_Nav">
+                <li><a class="active" href="#">Docs</a></li>   
+                <li><a class="active" href="#">Example</a></li>   
+            </ul>
+        </nav>
+
+
+
+        <!-- Sidebar , list of all the component -->
+        <div class="doc_Contianer">
```
Comment (rajkishancode) [inline Components/Input/input.html:33]: typo in class name

File: Components/Input/input.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/base16/ocean.min.css">
+</head>
+<body>
+    
+   
+    <div class="container">
+
+        <!-- Navbar , we can add mor in ul at right navbar -->
+
```
Comment (rajkishancode) [inline Components/Input/input.html:18]: remove extra space

File: Components/Input/input.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            </div>
+            <ul class="right_Nav">
+                <li><a class="active" href="#">Docs</a></li>   
+                <li><a class="active" href="#">Example</a></li>   
+            </ul>
+        </nav>
+
+
+
+        <!-- Sidebar , list of all the component -->
```
Comment (rajkishancode) [inline Components/Input/input.html:32]: remove extra space

File: Components/Input/input.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+            The Input tag is an empty element which only contains attributes.
+        </div>
+        <div class="divider"></div>
+
+    <!----label and input tag using input_Text--->
+        <div class="content_Head">
+            Text Box
+        </div>
+            <div class="content_Subtitle">
+                This is a simple input field feild to enter text.
```
Comment (rajkishancode) [inline Components/Input/input.html:132]: typo "feild"

File: Components/Input/input.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+                    </code>
+                </pre>
+            </div>
+            <div class="divider"></div>
+
+
+
+
+
+            <!----Different Validation textbox with different classes --->
```
Comment (rajkishancode) [inline Components/Input/input.html:154]: Please dont give space where not necessary.

File: Components/Input/input.html [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+        <div class="content_Head">
+            Validation
+        </div>
+            <div class="content_Subtitle">
+                This is a multiple input feild that response with variety.
```
Comment (rajkishancode) [inline Components/Input/input.html:159]: typo in field

**Review by rajkishancode — APPROVED** [review]
Left comment .Please do the needed changes  .

**Review by ianuragsharma — APPROVED** [review]
LGTM

**Pull request merged**